syntax = "proto3";

package grpc.tradeapi.v1;

option csharp_namespace = "Finam.TradeApi.Grpc.V1";

import "finam_grpc_client/models/proto/portfolios.proto";


service Portfolios {
  // Returns portfolio.
  // Возвращает портфель.
  rpc GetPortfolio(proto.tradeapi.v1.GetPortfolioRequest) returns (proto.tradeapi.v1.GetPortfolioResult);
}